src/views/vehicleServiceDetails/vehicleServicepage.jsx

```diff
@@ -1,429 +1,455 @@
+                                  <th scope="col">Description</th>
+                                </tr>
+                              </thead>
+                              <tbody>
+                                <tr>
+                                  <th scope="row">Lorem</th>
+                                  <td>
+                                    Lorem Ipsum is simply dummy text of the
+                                    printing and typesetting industry. Lorem
+                                  </td>
+                                </tr>
+                              </tbody>
+                            </Table>
+                          </Col>
+                        </Row>
+                      </Card>
+                    </Col>
+                  </Row>
+                </Card>
+                {/* Subscription Schedules */}
+                <Card className="shadow mt-5" body>
+                  <h3 className="mb-3">Subscription Schedules</h3>
+                  <Table className="align-items-center table-flush" responsive>
+                    <thead className="thead-light">
+                      <tr>
+                        <th scope="col">Service Date</th>
+                        <th scope="col" style={{ textAlign: 'center' }}>
+                          History
+                        </th>
+                        <th scope="col" style={{ textAlign: 'center' }}>
+                          Status
+                        </th>
+
+                        <th scope="col" />
+                      </tr>
+                    </thead>
+                    <tbody>
+                      <tr>
+                        <th scope="row">20-10-2019</th>
+                        <td style={{ textAlign: 'center' }}>
+                          <Button
+                            color="link"
+                            onClick={() => this.toggleModal('defaultModal')}
+                          >
+                            History link
+                          </Button>
+                        </td>
+                        <td style={{ textAlign: 'center' }}>Services Status</td>
+
+                        <td className="text-right">
+                          <UncontrolledDropdown>
+                            <DropdownToggle
+                              className="btn-icon-only text-light"
+                              href="#pablo"
+                              role="button"
+                              size="sm"
+                              color=""
+                              onClick={e => e.preventDefault()}
+                            >
+                              <i className="fas fa-ellipsis-v" />
+                            </DropdownToggle>
+                            <DropdownMenu className="dropdown-menu-arrow" right>
+                              <DropdownItem
+                                href="#pablo"
+                                onClick={e => e.preventDefault()}
```

Comment:
const preventDefault = e => e.preventDefault()

add this const in helper files and import/reuse here

pls avoid inline arrow functions in render method